{
  try {
    _persistence.countByC_V(RandomTestUtil.nextLong(),RandomTestUtil.nextDouble());
    _persistence.countByC_V(0L,0D);
  }
 catch (  Exception e) {
    Assert.fail(e.getMessage());
  }
}
